tox.ini

```diff
@@ -48,19 +58,13 @@ whitelist_externals =
     bash
     git
 
-[testenv:py{py27-subprocess,py27,py35,py36,py37,py38,py39,27,35,36,37,38,39,310,311}-integration]
+[testenv:py{py27-subprocess,py27,py35,py36,py37,py38,py39,27,35,36,37,38,39,310,311}-{,pip20-,pip22-}integration]
 deps =
     pytest-xdist==1.34.0
     {[testenv]deps}
 commands =
+    {[_printenv]commands}
     pytest -n auto tests/integration {posargs:-vvs}
-setenv =
```

Comment:
A lesson we already know over in Pants. Users may use the build tool but they probably don't understand it. When I took up work on Pex after Wickman / Wilson I did not know tox. I still do not know tox as well as I should. I did enough to get by. I now, however, understand the tox "generative names" concept better and realized this was all un-needed dup.